lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -189,7 +189,9 @@ private String getAuthorizationHeader(boolean renew) throws AblyException {
 		if(auth.getAuthMethod() == AuthMethod.basic) {
 			authHeader = "Basic " + Base64Coder.encodeString(auth.getBasicCredentials());
 		} else {
-			auth.authorise(null, null, renew);
+			Auth.AuthOptions options = new Auth.AuthOptions();
```

Comment:
I just think there is no point in creating the AuthOptions and for it to be merged when you know it only contains an option that is already the default.